src/helpers/measureTitleHelper.ts

```diff
@@ -272,3 +272,33 @@ export function fillMissingTitles(
     const measureTitleProps = buildMeasureTitles(measureBucketItems, locale, maxArithmeticMeasureTitleLength);
     return updateVisualizationObjectTitles(mdObject, measureTitleProps);
 }
+
+function removeTitleFromMeasures(bucket: IBucket): IBucket {
+    return {
+        ...bucket,
+        items: bucket.items.map(bucketItem => {
+            if (VisualizationObject.isMeasure(bucketItem)) {
+                delete bucketItem.measure.title;
+            }
+            return bucketItem;
+        }),
+    };
+}
+
+/**
+ * The function ignores the titles of the measures.
+ *
+ * The returned measures have no titles.
+ *
+ * @param {VisualizationObject.IVisualizationObjectContent} mdObject - metadata object that must be processed.
+ *
+ * @returns {VisualizationObject.IVisualizationObjectContent}
+ *
+ * @internal
+ */
+export function ignoreTitles(mdObject: IVisualizationObjectContent): IVisualizationObjectContent {
+    return {
+        ...mdObject,
+        buckets: mdObject.buckets.map(bucket => removeTitleFromMeasures(bucket)),
```

Comment:
Please add type for`bucket` and return type

Reply:
Fixed!